### ISSUE TITLE
Move Stock to Search

### ISSUE THREAD

**Issue by chadrolfs** [adobe-photoshop/spaces-design#2442]
Issue brought to you by Steve Guilhamet:
Library in DS includes Stock icon, it has been removed in classic (because it is part of Search widget now). Not sure what’s expected for DS, since you’re search is not the same as Library Search

**Comment by iwehrman** [comment]
I don't think this is a bug. We don't necessarily intend to provide the same UX as standard Photoshop.

**Comment by placegraphichere** [comment]
Adding enhancement as we'll probably need to do some tweaks to the search UI to handle the glut of images and content for this. We'll also need a way to visualize the stock (obviously) which we don't do for anything else in search.

**Comment by placegraphichere** [comment]
this is an older issue, but I would perhaps propose a couple options here:

1. Stock icon from Libraries panel would launch into our App-Search with Stock pre-selected. 
2. Remove stock icon from Libraries panel in favor of a better placement. This would launch into our App-Search with Stock pre-selected. 
3. Close issue and leave this as is (we'll have Stock in search soon)

**Comment by iwehrman** [comment]
I vote for 1).
